code/game/objects/items/weapons/storage/toolbox.dm

```diff
@@ -133,12 +133,13 @@
 	name = "brass box"
 	desc = "A huge brass box with several indentations in its surface."
 	icon_state = "brassbox"
+	item_state = null

```

Comment:
oh hey it's just a guy running down the hall i shouldnt ready my weapon he's unarmed and shi OH GOD HE HAS AN INVISIBLE BOX OF MADE OUT OF FUCKING BRASS